lib/reverse_sentence.rb

```diff
@@ -1,6 +1,31 @@
+#require "pry"
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: o(n)
+# Space complexity: o(1)
```

Comment:
You are doing `my_sentence.split` which creates a new array of equal size to the string. So... This is O(n) space complexity.

You were asked to do this in place.